vuln/npm/419.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 419,
+  "title": "Path Traversal",
+  "overview": "`superstatic` is vulnerable to path traversal on Windows and some Node.js versions",
```

Comment:
This is unclear.

It's `Windows || some Node.js versions`, while the overview could be read as `Windows combined with some Node.js versions`.

Given that this field could be multi-line and pretty big (it includes POCs with JS code for some entries), I propose something like this:

```md
`superstatic` is vulnerable to path traversal on Windows.

Additionally, it's vulnerable to path traversal on other platforms combined with certain Node.js versions which erroneously normalize `\` to `/` in paths on all platforms (a known example being Node.js v9.9.0).
```